config/routes.js

```diff
@@ -79,8 +76,28 @@ module.exports.routes = {
         'CheckUploadedDocumentsController.addDocsToDBHandler',
     '/handle-service-choice': 'ApplicationTypeController.handleServiceChoice',
 
+    // eligibility questions
+    '/eligibility-question-one':
+        'EAppEligibilityQuestionsController.renderQuestionOne',
+    '/handle-question-one-answer':
+        'EAppEligibilityQuestionsController.handleQuestionOneAnswer',
+    '/eligibility-question-two':
+        'EAppEligibilityQuestionsController.renderQuestionTwo',
+    '/handle-question-two-answer':
+        'EAppEligibilityQuestionsController.handleQuestionTwoAnswer',
+    '/eligibility-question-three':
+        'EAppEligibilityQuestionsController.renderQuestionThree',
+    '/handle-question-three-answer':
+        'EAppEligibilityQuestionsController.handleQuestionThreeAnswer',
```

Comment:
I actually didn't know you could use REST verbs in sailsjs because I hadn't seen it in this project but looks like you can. Can't imagine why they weren't used before 🤔